### ISSUE TITLE
The game freezes while loading files

### ISSUE THREAD

**Issue by GaborVarga239** [leezer3/OpenBVE#944]
### Description

The game freezes while loading files, it freezes with all maps, but i provided one example

### Reproduction

1.Select map and train, then press start
2. Game freezes

## Route

http://www.bvemetro.hu/  ("budapesti K-Ny-i metróvonal (2-es metró)")

## Train

https://vargazalanbve.webnode.hu/jarmuvek/   (The second train on the website)

## Logs

[openBVE Bug Report16_09_2023.zip](https://github.com/leezer3/OpenBVE/files/12640121/openBVE.Bug.Report16_09_2023.zip)

Logs may be accessed through the 'Report Problem' button on the bottom left of the screen.
Please provide a log, and a crash-log if the game actually crashes.

### Related information 

* Operating system - Arch Linux, wayland
* Method of control - keyboard

**Comment by leezer3** [comment]
Your log shows its loaded both the route and train, but not got into game, which is a little odd.

Are you sure it isn't still working? The final step may take a minute or two depending on the route & disk speed.

Can you move the window at all without it whiting-out?
If you can, this shows the render loop is still running.

You might also want to try Object Viewer and see if this works. That'll tell us whether it's something fundamental with the rendering somehow, or main game related.

**Comment by leezer3** [comment]
I've just tried this combination on Debian Bookworm, which works OK.

There was one report of freezing on loading, which we traced back to the Radeon proprietary driver:

**Comment by GaborVarga239** [comment]
> Your log shows its loaded both the route and train, but not got into game, which is a little odd.
> 
> Are you sure it isn't still working? The final step may take a minute or two depending on the route & disk speed.
> 
> Can you move the window at all without it whiting-out? If you can, this shows the render loop is still running.
> 
> You might also want to try Object Viewer and see if this works. That'll tell us whether it's something fundamental with the rendering somehow, or main game related.

Yeah, it didn't load into the game because it froze while "loading train x%", so i had to kill the process. i waited for 10-20 minutes still didn't load (im on a nvme ssd, so the disk speed is around 2000MB/s), i can move the window without whiting it out, and im using the amdgpu driver, and is it the drivers, or the games fault that its freezin?

**Comment by leezer3** [comment]
Honest answer is that I don't know unfortunately, only that they're interacting in some way.
Without an AMD card here I can't really debug, and if its a driver issue probably not a lot I could do anyway.

I can't see it being the games fault directly, but in terms of total user base we're miniscule.

Using MESA, running under WINE or Windows should work in this case as unsatisfactory as this is.....
